WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/Package Creation/ShippingLabelCustomPackageFormViewModel.swift

```diff
@@ -13,26 +13,65 @@ final class ShippingLabelCustomPackageFormViewModel: ObservableObject {
 
     /// The name of the custom package
     ///
-    @Published var packageName: String = ""
+    @Published var packageName: String
+
+    /// The type of the custom package, set to `box` by default
+    ///
+    @Published var packageType: PackageType
 
     /// The length of the custom package
     ///
-    @Published var packageLength: String = "0"
+    @Published var packageLength: String
 
     /// The width of the custom package
     ///
-    @Published var packageWidth: String = "0"
+    @Published var packageWidth: String
 
     /// The height of the custom package
     ///
-    @Published var packageHeight: String = "0"
+    @Published var packageHeight: String
 
     /// The weight of the custom package when empty
     ///
-    @Published var emptyPackageWeight: String = "0"
+    @Published var emptyPackageWeight: String
+
+    init(lengthUnit: String? = ServiceLocator.shippingSettingsService.dimensionUnit,
+         weightUnit: String? = ServiceLocator.shippingSettingsService.weightUnit,
+         packageName: String = "",
+         packageType: PackageType = .box,
```

Comment:
Just a note that although the designs show a "Select Type" placeholder (rather than defaulting to "Box") for the package type, both web and Android behavior is to default to the "Box" type, so I follow that behavior here.